Web/BootStrap-Assignment2/BoostrapMenu.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>Bootstrap Menu</title>
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+		<link rel="stylesheet" href="css/bootstrap.min.css" type="text/css">
+		<link rel="stylesheet" href="css/style.css" type="text/css">
+		<script type="text/javascript" src="js/bootstrap.min.js"></script>
+	</head>
+<body>
+<nav class="navbar navbar-inverse" role="navigation">
+ <div class="container-fluid">
+   <div class="navbar-header">
+   <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#myNavbar">
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>                        
+      </button>
+     <a class="navbar-brand" href="#">Metacube</a>
+   </div>
+  <div>
+     <ul class="nav navbar-nav visible-lg visible-md">
+       <li><a href="#">Home</a></li>
+       <li><a href="#">About</a></li>
+       <li><a href="#">Contact Us</a></li>
+       <li><a href="#">Logout</a></li>
+	   <li><a href="#">Movie Review</a></li>
+	   <li><a href="#">Movie Form</a></li>
+		<li class="dropdown">
+         <a class="dropdown-toggle" data-toggle="dropdown" href="#">CSS<span class="caret"></span></a>
+         <ul class="dropdown-menu">
+           <li><a href="#">Button</a></li>
+           <li><a href="#">Panel</a></li>
+           <li><a href="#">Grid</a></li>    
+			<li><a href="#">Link</a></li>
+           <li><a href="#">Pagination</a></li>
+           <li><a href="#">Progress Bar</a></li>     		   
+         </ul>
+       </li>
+	   </ul>
+		<ul class="nav navbar-nav navbar-right">
+		  <li>
+			<form class="navbar-form" role="search"> 
+				<div class="input-group"> 
+					<input type="text" class="form-control" placeholder="Search this site" name="srch-term" id="srch-term">
+					 <div class="input-group-btn">
+						 <button class="btn btn-default" type="submit">
+						 <i class="glyphicon glyphicon-search"></i>
```

Comment:
1.Search icon is not proper use some padding-top and bottom so that it's look like same height of text box
2. for small screen size devices  the placeholder of text box is not showing properly for small size device use small font of placeholder